src/components/List/List.md

```diff
@@ -28,6 +41,9 @@
 ```
 
 ### Inline List
+
+Use the *inline* `variant` to to arrange list items horizontally (inline).
```

Comment:
Looking below, *inline* is an `orientation`, not a `variant`

Reply:
Fixed